client/src/app/components/client-gridview/client-gridview.component.html

```diff
@@ -0,0 +1,18 @@
+<table>
+    <tr>
+        <th>Username</th>
+        <th>First Name</th>
+        <th>Last Name</th>
+        <th>Role</th>
+        <th>Is Active</th>
+        <th>E-mail</th>
+    </tr>
+    <tr *ngFor="let pers of users">
+        <td>{{pers.user_name}}</td>
+        <td>{{pers.first_name}}</td>
+        <td>{{pers.last_name}}</td>
+        <td>{{pers.role}}</td>
+        <td>{{pers.is_active}}</td>
+        <td>{{pers.email}}</td>
```

Comment:
And this will be handled in the backend by returning to the frontend only the required fields ( so for this purpose in the backend we would have a specific serializer for the view that just returns the fields that we want)

Reply:
if anything for now, let's not hard code the user's values. 
Just iterate over the user's values so that it can be dynamic for whatever fields are returned by the API for a user

Reply:
handled in the latest commit a3e471f6234310f1747af74339cb06ddb48ebe1d